src/sha1.c

```diff
@@ -117,41 +117,51 @@ int SHA1Reset(SHA1Context *context)
 *          the message.
 *      length: [in]
 *          The length of the message in message_array
+*          Max length can be 64 bits
```

Comment:
I don't get what this commit says

Reply:
Removed, meant to say max value of length